test/spec/comments.wast

```diff
@@ -35,10 +35,6 @@ module;;comment
 
 (;comment""";)
 
-(;Heiße Würstchen;)
-
-(;���������������;)
-
```

Comment:
Do we no longer support unicode in comments? That seems unfortunate.

Reply:
We do support it - there is no functional change in this PR. But we don't run the spec test for such comments. It seemed messy to get working, but maybe I missed a nice solution?

Reply:
I found a workaround, I think - just run that one test as a single file, avoiding the wast-splitting code which assumes the input is valid utf-8 (which that file is not). this way we still handle the test, but we don't run the binary format part (which is meaningless for a comments test anyhow)